applications/GeoMechanicsApplication/tests/cpp_tests/custom_constitutive/test_mohr_coulomb_with_tension_cutoff.cpp

```diff
@@ -31,6 +31,35 @@ using namespace std::string_literals;
 
 namespace Kratos::Testing
 {
+void FillParameters(Properties&                  properties,
+                    ConstitutiveLaw::Parameters& parameters,
+                    double                       FrictionAngle,
+                    double                       Cohesion,
+                    double                       DilatancyAngle,
+                    double                       TensileStrength)
+{
+    properties.SetValue(GEO_FRICTION_ANGLE, FrictionAngle);
+    properties.SetValue(GEO_COHESION, Cohesion);
+    properties.SetValue(GEO_DILATANCY_ANGLE, DilatancyAngle);
+    properties.SetValue(GEO_TENSILE_STRENGTH, TensileStrength);
+    parameters.SetMaterialProperties(properties);
+}
+
+Vector SetParametersAndLaw(ProcessInfo&                  process,
+                           Vector&                       cauchy_stress_vector,
+                           Vector&                       strain_vector,
+                           ConstitutiveLaw::Parameters&  parameters,
+                           MohrCoulombWithTensionCutOff& law)
```

Comment:
Please, change these parameter names such that they comply with the Kratos Style Guide:
```suggestion
                           Vector&                       rCauchyStressVector,
                           Vector&                       rStrainVector,
                           ConstitutiveLaw::Parameters&  rConstitutiveLawParameters,
                           MohrCoulombWithTensionCutOff& rConstitutiveLaw)
```

Reply:
fixed